### PR TITLE
feat: resize api

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

```ts
chart.resize({
  width: number;
});

chart.resize({
  height: number;
});

chart.resize({
  width: number;
  height: number;
});
```

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/store/options.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ const optionsData: StoreModule = {
       state.options = getOptionsBySize({ width, height }, state.originalOptions);
 
       this.dispatch('setChartSize', { width, height });
-      this.dispatch('updateTheme');
+      this.dispatch('initThemeState');
```
Comment (jungeun-cho) [inline src/store/options.ts:47]: `updateOptions`에서는 테마를 초기화하면안될것같아요.
옵션으로 테마정보도 바뀔수 있는건데 `initStoreState.options`로 계산되기 때문에 반영되지 않을꺼에요.

**Review by jungeun-cho — COMMENTED** [review]
[12/03] 확인했습니다~!
updateOptions에서 'initThemeState' 확인부탁드려요.
입력받은 options로 theme도 업데이트되어야할것같습니다.

**Review by lja1018 — APPROVED** [review]
[12/03] 리뷰완료
고생하셨습니다!

File: src/store/options.ts (hunk 1 above)
Comment (jung-han) [reply src/store/options.ts]: @jungeun-cho 
리뷰 반영되었습니다. 확인한번 부탁드릴게요~

**Review by jungeun-cho — APPROVED** [review]
[12/4] 수고하셨습니다. :)

**Pull request merged**